dirty_cat/datasets/tests/test_fetching.py

```diff
@@ -26,6 +26,7 @@ def get_test_data_dir() -> Path:
     return get_data_dir("tests")
 
 
+@pytest.mark.filterwarnings('ignore::UserWarning')
```

Comment:
I must admit that I don't really love silencing all the warnings on a test like this because it can silence legitimate warnings. I'd rather have at the right place (which can be in the test) a catch_warnings.